product-service/src/main/resources/docker-database.yml

```
@@ -25,4 +25,7 @@ spring:
     properties:
       hibernate:
         format_sql: true
-        show_sql: true
\ No newline at end of file
+        show_sql: true
+  batch:
+    jdbc:
+      initialize-schema: always
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)